website/pages/en/sunrise.mdx

```diff
@@ -0,0 +1,117 @@
+Once your subgraph has reached adequate curation signal and other Indexers begin supporting it, the upgrade Indexer will gradually taper off, allowing other Indexers to collect indexing rewards and query fees.
+
+### Should I host my own indexing infrastructure?
+
+Running infrastructure for your own project is [significantly more resource intensive](/network/benefits/) when compared to using The Graph Network.
+
+Additionally, The Graph Network is significantly more robust, reliable, and cost-efficient than anything provided by a single organization or team. Hundreds of independent Indexers around the world power The Graph Network, ensuring safety, security and redundancy.
+
+That being said, if you’re still interested in running a [Graph Node](https://github.com/graphprotocol/graph-node), consider joining The Graph Network [as an Indexer](https://thegraph.com/blog/how-to-become-indexer/) to earn indexing rewards and query fees by serving data on your subgraph and others.
+
+### Should I use a centralized indexing provider?
+
+If you are building in web2, you should. If you are building in web3, the moment you use a centralized indexing provider, you are giving them control of your dapp and data. The Graph’s decentralized network offers [superior quality of service](https://thegraph.com/blog/qos-the-graph-network/), reliability with unbeatable uptime thanks to node redundancy, as well as significantly [lower costs](/network/benefits/), and you won’t be held hostage at the data layer.
+
+With The Graph Network, your subgraph is public and anyone can query it openly, which increases the usage and network effects of your dapp. With a centralized indexing solution, the subgraph is private to the centralized provider.
+
+Here's a detailed breakdown of the benefits of The Graph over centralized hosting:
+
+Resilience and Redundancy: Decentralized systems are inherently more robust and resilient due to their distributed nature. Data isn't stored on a single server or location. Instead, it's served by hundreds of independent Indexers around the globe. This reduces the risk of data loss or service interruptions if one node fails, leading to exceptional uptime (99.99%).
+
+Quality of Service: In addition to the impressive uptime, The Graph Network features a ~106ms median query speed (latency), and higher query success rates compared to hosted alternatives. Read more in [this blog](https://thegraph.com/blog/qos-the-graph-network/).
+
+Censorship Resistance: Centralized systems can become targets for censorship, either through regulatory pressures or network attacks. In contrast, decentralized systems, due to their dispersed architecture, are much harder to censor, ensuring continuous data availability.
+
+Transparency and Trust: Decentralized systems operate openly, enabling anyone to independently verify the data. This transparency builds trust among network participants, as they can verify the system's integrity without relying on a central authority.
```

Comment:
Should the different points listed above ("Resilience and Redundancy", "Quality of Service", etc.) be...
- Bold?
- Headings? (####)
- Preceded by bullet points?

Reply:
Yes, good catch. Thank you! Bold + bullet points looks best locally